antarest/study/storage/variantstudy/model/command/update_binding_constraint.py

```diff
@@ -1,22 +1,108 @@
 import json
-from typing import Any, Dict, List, Mapping, Optional, Tuple
+import typing as t
 
 from antarest.core.model import JSON
 from antarest.matrixstore.model import MatrixData
-from antarest.study.storage.rawstudy.model.filesystem.config.binding_constraint import BindingConstraintFrequency
+from antarest.study.storage.rawstudy.model.filesystem.config.binding_constraint import (
+    BindingConstraintFrequency,
+    BindingConstraintOperator,
+)
 from antarest.study.storage.rawstudy.model.filesystem.config.model import FileStudyTreeConfig
 from antarest.study.storage.rawstudy.model.filesystem.factory import FileStudy
+from antarest.study.storage.rawstudy.model.filesystem.lazy_node import LazyNode
 from antarest.study.storage.variantstudy.model.command.common import CommandName, CommandOutput
 from antarest.study.storage.variantstudy.model.command.create_binding_constraint import (
     DEFAULT_GROUP,
-    TERM_MATRICES,
     AbstractBindingConstraintCommand,
+    TermMatrices,
     create_binding_constraint_config,
 )
 from antarest.study.storage.variantstudy.model.command.icommand import MATCH_SIGNATURE_SEPARATOR, ICommand
 from antarest.study.storage.variantstudy.model.model import CommandDTO
 
-MatrixType = List[List[MatrixData]]
+MatrixType = t.List[t.List[MatrixData]]
+
+ALIAS_OPERATOR_MAP = {
+    BindingConstraintOperator.EQUAL: "eq",
+    BindingConstraintOperator.LESS: "lt",
+    BindingConstraintOperator.GREATER: "gt",
+}
+
+HANDLED_OPERATORS = [
```

Comment:
This doesn't make sense, you should remove this

Reply:
Yeah I removed it in side of the corresponding function

Reply:
`_update_matrices_names`

Reply:
@sylvlecl we have different opinion on the subject with Mohamed so we would like to have yours :)

IMO we should remove this list and the associated check as it's not possible to have other operators since it's an enum.

On the other hand, Mohamed says it will be useful if someone wants to add another value to the Enum as it would crash (with my solution it won't crash here, it will enter the else and do something it shouldn't) and force the developer to adapt this code.